conan/tools/cmake/base.py

```diff
@@ -143,6 +149,15 @@ def __init__(self, conanfile, **kwargs):
 
         self.build_type = None
 
+        self.find_package_prefer_config = \
+            conanfile.conf["tools.cmake.cmaketoolchain"].find_package_prefer_config
+        if self.find_package_prefer_config is not None:
+            if isinstance(self.find_package_prefer_config, six.string_types):
+                self.find_package_prefer_config = self.find_package_prefer_config == "True"
+            elif isinstance(self.find_package_prefer_config, six.integer_types):
+                self.find_package_prefer_config = bool(self.find_package_prefer_config)
+            self.find_package_prefer_config = "ON" if self.find_package_prefer_config else "OFF"
+
```

Comment:
two notes:
- right now, `conanfile.conf` always returns string, so it returns string value `True` instead of boolean `True`. I am not sure if it will be always the same, so I check type for the safety
- I believe it should be a function (just like [to_meson_value](https://github.com/conan-io/conan/blob/639450053a0359af5a757b3617a82a2ece2f7cf1/conan/tools/meson/toolchain.py#L77)), but I guess we will do it to address #8638  anyway